Chummer/Backend/Equipment/Cyberware.cs

```diff
@@ -509,6 +510,10 @@ private void GearChildrenOnCollectionChanged(object sender, NotifyCollectionChan
             if (!objXmlCyberware.TryGetMultiLineStringFieldQuickly("altnotes", ref _strNotes))
                 objXmlCyberware.TryGetMultiLineStringFieldQuickly("notes", ref _strNotes);
 
+            String sNotesColor = ColorTranslator.ToHtml(ColorManager.HasNotesColor);
+            objXmlCyberware.TryGetStringFieldQuickly("notesColor", ref sNotesColor);
+            _colNotes = ColorTranslator.FromHtml(sNotesColor);
+
```

Comment:
Is this necessary in a method on/near creation of gear? Would there need to be data-definable colors for specific gear?